### PR TITLE
Test for SRTomo class

### PR DESCRIPTION
**Pull request by victortxa**
Making a test for SRTomo class. Maybe, also changes in docstring of this class to an adjustment.

### Checklist:

- [x] Make tests for new code (at least 80% coverage)
- [x] Create/update docstrings
- [x] Include relevant equations and citations in docstrings
- [x] Docstrings follow the style conventions
- [x] Code follows PEP8 style conventions
- [x] Code and docs have been spellchecked
- [x] Include new dependencies in `doc/install.rst`, `requirements.txt`, `environment.yml`, `ci/requirements-conda.txt` and `ci/requirements-pip.txt`.
- [x] Documentation builds properly (run `cd doc; make` locally)
- [x] Changelog entry (leave for last to avoid conflicts)
- [x] First-time contributor? Add yourself to `doc/contributors.rst` (leave for last to avoid conflicts)

### CODE REVIEW

**Comment by leouieda** [comment]
@victortxa thanks for doing this! 

To test the `predicted` method, you can set the `p_` attribute by hand to the correct value and compare the result of `tomo.predicted()` with the forward modeled data.

Also, be careful of PEP8 issues in your code (particularly missing spaces after commas). Use `make pep8` to check your code. Travis will catch this as well.

**Comment by victortxa** [comment]
@leouieda  Thank you! 
A doubt:  `p_` is an attribute of SRTomo, so should I call `SRTomo.p_=list_of_p_values` to set the values ?

**Comment by leouieda** [comment]
Something like:

    tomo = SRTomo(...)
    tomo.p_ = some_array

File: fatiando/seismic/tests/test_seismic_srtomo.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+def test_general():
+    """
+    General test of the class SRTomo, as it is in the docs of this class.
+    """
+    model = SquareMesh((0, 10, 0, 10), shape=(2, 1), props={'vp': [2., 5.]})
+    src = (5, 0)
+    srcs = [src, src]
+    recs = [(0, 0), (5, 10)]
+    ttimes = ttime2d.straight(model, 'vp', srcs, recs)
+    tomo = srtomo.SRTomo(ttimes, srcs, recs, model)
```
Comment (leouieda) [inline fatiando/seismic/tests/test_seismic_srtomo.py]: Please create a `mesh` object using `SquareMesh` to use with the tomo class. The `model` object is mutable and you won't have it if you're not running a synthetic data test.

File: fatiando/seismic/tests/test_seismic_srtomo.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import numpy as np
```
Comment (leouieda) [inline fatiando/seismic/tests/test_seismic_srtomo.py]: On the first line, use the following imports for future Python 3 compatibility:

    from __future__ import division, print_function
    from future.builtins import range

This way you don't need to put `.` in all your numbers "`1./2.`"

**Comment by leouieda** [comment]
@victortxa looks good so far! It seems that line 117 is the one not covered in the tests (it's that check if the predicted is a 1D numpy array). Could you try removing that `if` statement? If the tests pass and the [gallery plot](https://github.com/fatiando/fatiando/blob/master/gallery/5-seismic/srtomo_regularized.py) runs as expected then I think it should be safe.

After that, I think all that is left is building the docs and checking is all docstrings and gallery plots are working correctly.

**Comment by victortxa** [comment]
I'm trying to run tests in my laptop, but I need to fix the installations (a lot of things failing here).

**Comment by victortxa** [comment]
@leouieda I think it's all working now. I removed the example from the 'docstrings' of the function (take a look if you agree).

**Comment by leouieda** [comment]
@victortxa yep, it's all looking good! Thanks!

Could you please merge in the latest commits from master and create the changelog entry? Also, build the docs locally and check that everything is OK, particularly the SRTomo docs and the changelog.

File: doc/changelog.rst [hunk 3]
```diff
@@ -10,6 +10,8 @@ Version 0.5
 
 **Changes**:
 
+* Add test for SRTomo class.
```
Comment (leouieda) [inline doc/changelog.rst]: Please be more detailed, for example:

    Implement more unit tests for the ``fatiando.seismic.srtomo`` module. Reached 100% test coverage.

**Comment by leouieda** [comment]
@victortxa OK, perfect! Sorry, you'll need to merge in master again because #319 was just merged. After that, let me know and I'll merge this.

**Comment by victortxa** [comment]
@leouieda After checks pass, I think it's done.

**Comment by leouieda** [comment]
Merged!

**Pull request merged**